README.md

```diff
@@ -86,7 +60,8 @@ All the exception and log messages in this project are consistent with the follo
 When contributing to this project, please follow these guidelines.
 
 ## License
+
 ScalarDB is dual-licensed under both the Apache 2.0 License (found in the LICENSE file in the root directory) and a commercial license.
 You may select, at your option, one of the above-listed licenses.
 The commercial license includes several enterprise-grade features such as ScalarDB Server, management tools, and declarative query interfaces like GraphQL and SQL interfaces.
```

Comment:
Since ScalarDB Server is deprecated, how about if we mention ScalarDB Cluster instead?

```suggestion
The commercial license includes several enterprise-grade features such as ScalarDB Cluster, management tools, and declarative query interfaces like GraphQL and SQL interfaces.
```